test/gzip.bats

```diff
@@ -0,0 +1,129 @@
+load helpers
+
+function setup() {
+    stacker_setup
+}
+
+function teardown() {
+    cleanup
+}
+
+@test "import tar < 1M" {
+  mkdir -p folder1
+  truncate -s 512k  folder1/file1
+  tar cvf test.tar folder1
+  cat > stacker.yaml <<EOF
+tar:
+  from:
+    type: tar
+    url: test.tar
+EOF
+  stacker build
+  cat oci/index.json | jq .
+  m1=$(cat oci/index.json | jq .manifests[0].digest | sed  's/sha256://' | tr -d \")
+  cat oci/blobs/sha256/"$m1" | jq .
+  l1=$(cat oci/blobs/sha256/"$m1" | jq .layers[0].digest | sed  's/sha256://' | tr -d \")
+  [[ "$($SKOPEO --version)" =~ "skopeo version ${SKOPEO_VERSION}" ]] || {
+    echo "$SKOPEO --version should be ${SKOPEO_VERSION}"
+    exit 1
+  }
+  $SKOPEO copy --format=oci oci:oci:tar containers-storage:test:tar
+  $SKOPEO copy --format=oci containers-storage:test:tar oci:oci:test
+  cat oci/index.json | jq .
+  m2=$(cat oci/index.json | jq .manifests[1].digest | sed  's/sha256://' | tr -d \")
+  cat oci/blobs/sha256/"$m2" | jq .
+  l2=$(cat oci/blobs/sha256/"$m2" | jq .layers[0].digest | sed  's/sha256://' | tr -d \")
+  echo "$l1"
+  echo "$l2"
+  [ "$l1" = "$l2" ]
+}
```

Comment:
Can we make a single function which takes the test params?

```
test_copy_buffer_size()
   local buffer_size = "$1"
   rm -rf "folder1"
   mkdir folder1
   truncate -s "$buffer_size" folder1/file1
   tar cvf test.tar folder1
   cat > stacker.yaml <<EOF
   ....
   [ "$l1" = "$l2" ]
}
```

Then do

```
@test "import tar > 1M" test_copy_buffer_size "2M"
```

or incase the `@test` doesn't accept params we can

```
test_copy_buffer_2M() {
    test_copy_buffer_size "2M"
}

@test "import tar > 1M" test_copy_buffer_2M
```

This makes it easier to see where the variablity lies and makes adjusting
the test case simpler (fix in one spot)

Reply:
Let me check this

Reply:
Fixed.